Several Updates to FT Analysis

mlkhorn opened the pull request:
In this code I:
-Further clean my data removing incomplete years of CO2 flux data and negative CO2 flux values.
-Add some preliminary visualizations of my data.
-Summarize my data by calculating annual Rs, winter Rs, winter Rs contribution, mean annual T5, mean winter T5, mean annual SM5, and mean winter SM5 for each year of each dataset.

mlkhorn committed:
Several Updates to FT Analysis

In this code I:
-Further clean my data removing incomplete years of CO2 flux data and negative CO2 flux values.
-Add some preliminary visualizations of my data.
-Summarize my data by calculating annual Rs, winter Rs, winter Rs contribution, mean annual T5, mean winter T5, mean annual SM5, and mean winter SM5 for each year of each dataset.

bpbond commented on COSORE Winter Rs Analysis.Rmd:
```diff
@@ -112,45 +265,76 @@ for(i in 1:length(winterdata_names)){
+  summarise(avg_winter_Rs=mean(CSR_FLUX_CO2, na.rm= TRUE)) -> wRs_increment
+ 
+wRs_increment %>% 
+  mutate(winter_Rs=avg_winter_Rs*31.536*12.0107*0.25) -> winterdata_summary
```
This line could use a comment explaining what these constants are and, in general, what the conversion/calculation being done here is..

mlkhorn replied:
Yes, good point!

mlkhorn committed:
Commented Code

This is my ft script as of Friday. I wanted to add a few more comments before our meeting tomorrow but I ran into a bug I needed to fix. Planning on adding a few more comments here and there tomorrow, but for now I think it's good.

mlkhorn commented:
@bpbond @stephpenn1 I added the requested comment and some other advances last week. Had a little problem with one of my visualizations I tabled Friday night and figured out a couple minutes ago. English essay done, bug fixed, exciting meeting tomorrow morning... result: I am feeling like an all-star! :) See you tomorrow.

bpbond commented on COSORE Winter Rs Analysis.Rmd:
````diff
@@ -40,13 +43,18 @@ library(tibble)
 
 ```{r source-data-id-script}
+
+#This code pulls in data relevant to the desired analysis
````
Not a very informative comment. What data? What analysis? Be concise but clear.

mlkhorn committed:
Figures Added

This code adds a figure showing where the six sites with clean data I identified are located, a table summarizing annual and winter Rs data, and a figure summarizing the model diagnostics of the winter Rs contribution ANOVA.

stephpenn1 committed:
bar chart

mlkhorn committed:
Merge pull request #18 from mlkhorn/sp-figures

SP Figure Tweaks

mlkhorn committed:
Small Changes

Got rid of of some comments.

mlkhorn committed:
Merge branch 'ft-events' of https://github.com/mlkhorn/ftc_cosore into ft-events

bpbond commented:
Hi @mlkhorn just wondering, what's the status of this PR? If you're not going to merge it probably close.

mlkhorn committed:
Bar Graph Update

In this code I added three bar graphs showing winter Rs, annual Rs, and winter contribution.

stephpenn1 commented:
@mlkhorn please push up any final changes on this script so we can pull this into master
